doc/nrf/external_comp/nrf_cloud.rst

```diff
@@ -13,7 +13,11 @@ Overview
 ********
 
 nRF Cloud allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.
-It also helps your devices determine their location.
+FOTA can be used to update the device application, bootloader, and modem.
+The modem can be incrementally updated with a modem delta image.
+If the device has sufficiently large external flash storage, the modem can be entirely updated with a full modem image.
+It also helps your devices determine their location using assisted GPS (A-GPS) and predicted GPS (P-GPS).
```

Comment:
maybe remove this 'also' since there is another on the next line, and 'additionally' the line after that.